src/developer-testing/dto/create-admin.dto.ts

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray } from 'class-validator';
+
+export class CreateCddProviders {
```

Comment:
nit -> Add Dto to the class name and make sure the class name and file name are in sync :)